### ISSUE TITLE
Align styling of toggled actions and toggled buttons

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#200810]
We show a maximized editor group with a toggled icon:

<img width="275" alt="image" src="https://github.com/microsoft/vscode/assets/900690/780c305a-efd4-459a-b3be-7d6808cc53ab">

Meanwhile we have an existing style for when something is toggled:

<img width="223" alt="image" src="https://github.com/microsoft/vscode/assets/900690/acf5b98f-2b1b-4a9a-906e-785e4ba1dc8e">

I think the latter is much more explicit and inviting to click, so I think we should align the former to it.

//cc @joaomoreno

**Comment by gjsjohnmurray** [comment]
My PR #198979 is slightly related to this UX topic.

**Comment by benibenj** [comment]
Aligning seems good, but personally, I think the second option stands out a bit too much for actions in for example the tab bar. I think these are actions that you don't press often. They should indicate to the user that something is active, but not be too inviting to press. 

@hediet I think you introduced this toggled background for your diff setting icons. What are your thoughts on this?

**Comment by bpasero** [comment]
Just for comparison: XCode changes the icon when you maximize a group and they choose to show a prominent blue background color as well:

![image](https://github.com/microsoft/vscode/assets/900690/6f2cc1fd-69b8-4bb8-b3ce-dcaa8108d01f)

**Comment by bpasero** [comment]
Yes, works for me. I am with Ben that the less strong option would be preferable considering that you might have this toggled state on all the time.

**Comment by hawkticehurst** [comment]
Fixed via https://github.com/microsoft/vscode/pull/283569
